podcasts/Bookmarks/List/BookmarkListViewModel.swift

```diff
@@ -4,9 +4,18 @@ import PocketCastsDataModel
 protocol BookmarkListRouter: AnyObject {
     func bookmarkPlay(_ bookmark: Bookmark)
     func bookmarkEdit(_ bookmark: Bookmark)
+    func dismissBookmarksList()
+
+    var alertController: UIViewController? { get }
```

Comment:
This isn't really an alert controller, but more of a `UIViewController` in which an actual alert controller will be presented, right?

I think a more generic name + a comment can help clarify that.